Portfolio/index.html

```diff
@@ -1,177 +1,205 @@
-        <ul>
-            <li>HTML</li>
-            <li>CSS</li>
-            <li>JavaScript</li>
-            <li>Python</li>
-            <li>Photoshop</li>
-        </ul>
+        <h4>
+            <div>Name:John Doe</div>
+            <div>Age:24</div>
+            <div>Experience: 2years</div>
+        </h4>
+        </div>
     </section>
 
-    <section class="Exper">
-        <h2>Experience</h2>
-        <p>I have over fresher of experience. Throughout my career, I've worked on projects related to web development.</p>
-    </section>
+    <!-- skills -->
+    <section class="skills" id="skills">
 
-    <section>
-        <h2>Education</h2>
-        <p>I graduated from KIT College of engineering, Kolhapur with a degree in Btech. Additionally, I've completed courses in computer and Science.</p>
-    </section>
+        <h1>SKILLS</h1>
 
-    <section>
-        <h2>Contact Me</h2>
-        <form action="mailto:omparthchougale2182002@gmail.com" method="GET">
-            <label for="subject">Subject:</label><br>
-            <input type="text" id="subject" name="subject"><br>
-            <label for="message">Message:</label><br>
-            <textarea id="message" name="body"></textarea><br>
-            <input type="submit" value="Send">
-        </form>
-    </section>
+        <h4> <img src="/html.icon.png" alt="Css Image" width="30px" height="30px">
+            <div class="skilltext">HTML</div>
+        </h4>
 
-    <footer>
-        <h2>Connect with Me</h2>
-        <p>Feel free to connect with me on:</p>
-        <ul>
-            <li><a href="https://www.linkedin.com/in/omparth-chougale-582b89245/" target="_blank">LinkedIn</a></li>
-            <li><a href="https://twitter.com/Omparth2" target="_blank">Twitter</a></li>
-            <li><a href="https://github.com/omparth" target="_blank">GitHub</a></li>
-        </ul>
-    </footer>
-    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
-    <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="task1.css">
-    <title>About Me</title>
-    <style>
-    
-    </style>
-</head>
-<body>
+        <h4> <img src="/css.icon.png" alt="Css Image" width="30px" height="30px">
+            <div class="skilltext">CSS</div>
+        </h4>
 
-    <header>
-       
-        <h1>About Me</h1>
-    </header>
+        <h4> <img src="/js.icon.png" alt="Css Image" width="30px" height="30px">
+            <div class="skilltext">JavaScript</div>
+        </h4>
 
-    <section class="exp">
-      <img src="../public/johnDoe.png" alt="CSS image">    
-        <h2>Introduction</h2>
-        <p>Hello there! I'm Omparth Maruti Chougale, a passionate <b>Software Engineer</b>  with a keen interest to watch Anime. I love to Code and to develop Websites.</p>
-    </section>
+        <h4> <img src="/ts.icon.png" alt="Css Image" width="30px" height="30px">

```

Comment:
The closing </div> tag at line 150 is misplaced. It should be moved after line 149.